startup/10-machine.py

```diff
@@ -95,14 +95,14 @@ def move(self, *args, moved_cb=None, **kwargs):
         if moved_cb is not None:
 
             @functools.wraps(moved_cb)
-            def inner(obj=None):
+            def inner_move(status, obj=None):
                 if obj is not None:
                     obj = self
-                return moved_cb(obj=obj)
+                return moved_cb(status, obj=obj)
```

Comment:
@tacaswell, will this work for ophyd 1.3.3 and older?

The description of the kwarg in https://github.com/bluesky/ophyd/blob/eb55f56b8641b3d5703c8179cf3023669d422593/ophyd/pseudopos.py#L810-L813 for newer ophyd seems to be incorrect.

Reply:
It was failing on the older conda envs, as suspected. But it's fine, we will do our best to run on 2020-2.0 conda env.